KoikatuGameplayMod/KoikatuGameplayMod.cs

```diff
@@ -70,7 +72,7 @@ private void Start()
 
             SceneManager.sceneLoaded += (arg0, mode) =>
             {
-                if (arg0.name != "MyRoom" || Singleton<Scene>.Instance.LoadSceneName == "H")
+                if (arg0.name != "MyRoom" || Singleton<Scene>.Instance.LoadSceneName == "H" || Application.productName == GameProcessNameVR)
```

Comment:
Save Application.productName == GameProcessNameVR to a field and use it everywhere instead. That way you can compare to the GameProcessNameVRSteam easily.

Reply:
Normally I'd do that for my own plugins, but I wasn't sure about your preference for implementing such a field.
Such as the class to contain the field and other stylistic choices.

Reply:
In this case the comparison always returns the same value, so I prefer to calculate it once and store the variable somewhere.